java-client/src/main/java/org/opensearch/client/opensearch/_types/ErrorResponse.java

```diff
@@ -62,14 +62,14 @@ private enum Kind {
 
     private final ErrorCause error;
 
-    private final int status;
```

Comment:
@VachaShah hm I believe `status` should be always present, we may have an issue on the server side?

Reply:
@reta I dug through the server side and the server itself does not return `status` for msearch template requests. (see additional context section in https://github.com/opensearch-project/opensearch-java/issues/708).

Reply:
The server does not set a status for MultiSearchTemplateResponse (https://github.com/opensearch-project/OpenSearch/blob/main/modules/lang-mustache/src/main/java/org/opensearch/script/mustache/MultiSearchTemplateResponse.java#L169) whereas for MultiSearchResponse, it does https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/action/search/MultiSearchResponse.java#L195.

Reply:
Do you think this is something that should be fixed on the server side? I am not sure if there is any history behind not setting status for mutli search templates 🤔

Reply:
Thanks a lot for digging it in @VachaShah !

> Do you think this is something that should be fixed on the server side?

I believe so, the status field should always be set by the server.

Reply:
Got it, thank you @reta! I think we should still merge this fix though so that the users can use it with the current versions of OpenSearch, WDYT?

Reply:
Oh that's right, no objection

Reply:
Thank you! I will raise an issue on OpenSearch to add the status field for multi search templates.

Reply:
@reta I opened an issue on the server for this: https://github.com/opensearch-project/OpenSearch/issues/11133